drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -15,7 +15,7 @@ if ETH_SAM_GMAC
 config ETH_SAM_GMAC_QUEUES
 	int "Number of active hardware TX and RX queues"
 	default 1
-	range 1 $(dt_int_val,DT_INST_0_ATMEL_SAM_GMAC_NUM_QUEUES)
+	range 1 $(dt_node_int_prop_int,/soc/ethernet@40050088,num-queues)
```

Comment:
This might have been an oversight since `ethernet@40050088` is not universal. We need path to deal with this ...

Reply:
See #24256